tensorflow_model_optimization/g3doc/guide/pruning/index.md

```diff
@@ -41,6 +43,18 @@ It is on our roadmap to add support in the following areas:
 *   [Minimal Subclassed model support](https://github.com/tensorflow/model-optimization/issues/155)
 *   [Framework support for latency improvements](https://github.com/tensorflow/model-optimization/issues/173)
 
+## Structural pruning M by N
```

Comment:
The overview page is not for a specific pruning technique. This section doesn't look very appropriate to me being here, so what about moving this to the subpage (tutorial) if you want to keep it

Rather than this section, I think the experiment results / and a brief comment below the result about what 2x4 means / would be enough.